File: java-script/untitled.js
let random = (x) => {
	x ^= x << 13;
	x ^= x >> 17;
	x ^= x << 5;
	return ((x < 0 ? ~x + 1 : x) % 100) / 100;
};
let z = 546;
let a = [5, 3, 3, 5, 7, 8, 5, 4];
a.sort((x) => {
	z ^= z << 43;
	return random(z) - 0.5;
});
console.log(a);

File: termuxgui/untitled.py
import termuxgui as tg
import time
#import os

with tg.Connection() as c:
    wid = "78494ca9-cf15-485a-9db9-7f31c2ccaf30"
    # Create a remote layout
    rv = tg.RemoteViews(c)
    # Add a TextView
    tv = rv.addTextView()
    while True:
        #files = os.listdir()
        #files.sort()
        # Set the text
        rv.setText(tv, "abcdefghijklmnopqrstuvwxyz\n" * 5)
        # update the widget
        rv.updateWidget(wid)
        time.sleep(1)